### PR TITLE
Add html.manifest.id BCD

### PR DESCRIPTION
**Pull request by dawei-wang**
Fixes mdn/browser-compat-data#18192

### CODE REVIEW

File: html/manifest/id.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": "mirror",
+            "webview_android": {
+              "version_added": null
+            }
```
Comment (queengooborg) [inline html/manifest/id.json:37]:
```suggestion
            "webview_android": "mirror"
```

File: html/manifest/id.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
```
Comment (queengooborg) [inline html/manifest/id.json:33]:
```suggestion
            "safari_ios": "mirror",
```

File: html/manifest/id.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
```
Comment (queengooborg) [inline html/manifest/id.json:27]:
```suggestion
            "opera": "mirror",
            "opera_android": "mirror",
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thanks for your PR, and welcome to BCD!  The sources that are linked in the issue LGTM for Chrome and Firefox.  For the other Chromium forks, we can just mirror their data.

**Comment by dawei-wang** [comment]
@queengooborg Sounds good, I have committed the suggested changes.

**Review by queengooborg — APPROVED** [review]
Thank you, this is LGTM!

**Pull request merged**